app/views/cities/show.html.erb

```diff
@@ -1,6 +1,13 @@
 <h3> <%= @city.name %></h3>
 <img class='banner-img' src=<%= @city.photo%>>
-<%= button_to "Add New Tip", new_tip_path, method: "get"%>
-<% @city.tips.each do |tip|%>
-  <%= link_to tip.title, tip_path(tip) %> <br>
-<% end %>
+<br>
+<br>
+<div class="container">
+  <%= button_to "Add New Tip", new_tip_path, method: "get"%>
+  <% @city.tips.order(created_at: :desc).each do |tip|%>
```

Comment:
Is this the cities#show page or the tips#index page for a specific city? If you think of it as being a tips#index you may be able to move some of this logic into a different view, a view that only knows about "locations" and not "cities".